recipes/ecco_v4_py/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.1.6" %}
+
+package:
+  name: ecco_v4_py
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/e/ecco_v4_py/ecco_v4_py-{{ version }}.tar.gz
+  sha256: 419da92d36a739586f31c9b0bc132c1fd91b8cb33b5ef58064f64b2caa4a954b
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python
+    - pip
+    - setuptools
+  host:
+    - python
```

Comment:
Is this python3 only? When I look at the `setup.py` the metadata suggests so. If that's the case use `python  >=3.5` instead of just `python` under `host` and `run` (using whatever version of python that is actually minimally supported).

Reply:
Yes, thanks! This is something I'm squaring away at the moment.

Reply:
Ok this is an interesting one. I agree we should be specifying a python version, but the conda-forge linter seems to have a problem with non, noarch python builds specying a python version. It seems that we can't specify the `noarch: python` line since we require 2 binary files with the package.